u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AndroidKeyStoreAttestation.java

```diff
@@ -49,7 +49,10 @@ private AndroidKeyStoreAttestation(Integer keymasterVersion, byte[] attestationC
   }
 
   /**
-   * Parses the key description extension.  Expected format is:
+   * Parses the key description extension.  Note that this method only parses the description
+   * extension in the leaf cert.  It *does not* validate the certificate (or any chain).
```

Comment:
Yes, I intend to get there (validate the whole chain) soon.  There are at least two things that need to be done:
  * Validate the whole cert chain
  * Add UI elements that mention whether the chain verified

Do you want me to add TODOs for them?